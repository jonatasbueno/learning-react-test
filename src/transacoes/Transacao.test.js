import React from 'react';
import Transacao from './Transacao';
import { render } from '@testing-library/react';

describe('Componente Transação', () => {
  /**
   * O teste de snapshot ocorre pra gantir a renderização de todos lementos HTML
   */
  it('teste snapshot', () => {
    const { container } = render(<Transacao data="25/09/2021" tipo="deposito" valor='100' />);

    expect(container.firstChild).toMatchSnapshot();
  })
})