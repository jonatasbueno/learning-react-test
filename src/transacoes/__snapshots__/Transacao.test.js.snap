// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente Transação teste snapshot 1`] = `
<div
  class="transacao-container"
>
  <p>
    25/09/2021
  </p>
  <p>
    deposito
  </p>
  <p>
    R$ 
    100
  </p>
</div>
`;
